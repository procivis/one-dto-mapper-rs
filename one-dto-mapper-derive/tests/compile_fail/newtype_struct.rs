use one_dto_mapper_derive::From;

struct Tuple2(u32);

#[derive(From)]
#[from(Tuple2)]
struct Tuple(u32);

fn main() {}
