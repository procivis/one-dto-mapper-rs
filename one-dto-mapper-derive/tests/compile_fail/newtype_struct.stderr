error: Unsupported shape `one unnamed field`. Expected named fields.
 --> tests/compile_fail/newtype_struct.rs:5:10
  |
5 | #[derive(From)]
  |          ^^^^
  |
  = note: this error originates in the derive macro `From` (in Nightly builds, run with -Z macro-backtrace for more info)
